icu4c/source/i18n/numparse_types.h

```diff
@@ -64,8 +64,9 @@ class CompactUnicodeString {
         fBuffer[0] = 0;
     }
 
-    CompactUnicodeString(const UnicodeString& text)
-            : fBuffer(text.length() + 1) {
+    CompactUnicodeString(const UnicodeString& text, UErrorCode& status)
+        : fBuffer(text.length() + 1, status) {
```

Comment:
Needs indentation. My preference:
```
    CompactUnicodeString(const UnicodeString& text, UErrorCode& status) :
            fBuffer(text.length() + 1, status) {
        if (U_FAILURE(status)) { return; }
        ...
```

Reply:
Can I go with the old formatting of this line, i.e. going for minimal diffs? (I've done this already.)

Maybe a good time to discuss https://github.com/unicode-org/icu/pull/1143 - Younies and I: are taking guidance from clang-format for new code. I manually override to keep things consistent with other code in the same file, but for new files I basically just follow clang-format's recommendations.

I wrote a bunch more here, but I'm moving that to #1143.

Reply:
If given a choice, I prefer the `:` on the line below the constructor signature, but I most prefer to defer to clang-tidy.